common/changes/@rushstack/heft/ianc-webpack-plugin_2021-04-06-03-03.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "@rushstack/heft",
+      "comment": "(BREAKING) Move Webpack functionality into its own package (@rushstack/heft-webpack4-plugin).",
+      "type": "minor"
+    }
```

Comment:
Also, can Heft print a warning if it sees a `webpack.config.js` but the project is not loading `@rushstack/heft-webpack___-plugin`?